src/composables/getCollections.js

```diff
@@ -0,0 +1,32 @@
+import { ref } from 'vue';
+import { projectFirestore } from '../firebase/config';
+
+const getCollection = (collection) => {
```

Comment:
Shouldn't be async function? Consider cause you make a request to firestore so this is asynchronous operation.

Reply:
i explain it yesterday, i prefer to not change it.